src/sentry/static/sentry/app/views/settings/account/accountSecurity/components/removeConfirm.tsx

```diff
@@ -16,6 +18,12 @@ const message = (
   </React.Fragment>
 );
 
-const RemoveConfirm = props => <Confirm {...props} message={message} />;
+class RemoveConfirm extends React.Component<Props> {
+  static defaultProps = Confirm.defaultProps;
```

Comment:
Unfortuantely this frustratingly does need to be a class because of how we use `defaultProps` on the `Confirm` component.

We'll probably have to talk about this at the TSC soon

Reply:
oh this is weird ... what is the error you get?